CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file. This library adheres to a versioning policy described in [the README](./README.md#versioning). The public API of this library consists of the functions exported in [h3core.js](./lib/h3core.js).
 
 ## [Unreleased]
+### Fixed
+- Fixed `h3IsValid` returning true on certain edge cases (#81)
+### Changed
+- Updated the core library to 3.6.3 (#81)
```

Comment:
nit: do we want to mention what was changed in the core library?

Reply:
I say unless we come up with a good way to auto-populate that, it's a bunch of make-work and not super valuable (those interested can look at the core library CHANGELOG).

Reply:
Added a short note. I think it's worthwhile if the changes are substantive, since most consumers will only look at the binding notes and may not realize they need to look further to understand changes.